common/src/main/scala/org/genivi/sota/marshalling/CirceInstances.scala

```diff
@@ -58,29 +55,22 @@ trait CirceInstances {
   implicit val uuidEncoder : Encoder[UUID] = Encoder[String].contramap(_.toString)
   implicit val uuidDecoder : Decoder[UUID] = Decoder[String].map(UUID.fromString)
 
-  implicit val dateTimeEncoder : Encoder[DateTime] =
-    Encoder.instance[DateTime]( x =>  Json.fromString( ISODateTimeFormat.dateTime().print(x)) )
+  implicit val dateTimeEncoder : Encoder[Instant] =
+    Encoder.instance[Instant]( x =>  Json.fromString( x.toString) )
 
-  implicit val dateTimeDecoder : Decoder[DateTime] = Decoder.instance { c =>
+  implicit val dateTimeDecoder : Decoder[Instant] = Decoder.instance { c =>
     c.focus.asString match {
       case None       => Xor.left(DecodingFailure("DataTime", c.history))
       case Some(date) =>
-        val parsers = List(ISODateTimeFormat.dateTimeNoMillis(), ISODateTimeFormat.dateTime())
-        tryParsers(date, parsers, DecodingFailure("DateTime", c.history))
+        tryParser(date, DecodingFailure("DateTime", c.history))
     }
   }
 
-  @tailrec
-  private def tryParsers(string: String, parsers: List[DateTimeFormatter],
-                         error: DecodingFailure): Xor[DecodingFailure, DateTime] = {
-    parsers match {
-      case parser :: otherParsers =>
-        try { Xor.right(parser.parseDateTime(string)) }
-        catch {
-          case t: IllegalArgumentException =>
-            tryParsers(string, otherParsers, error)
-        }
-      case Nil => Xor.left(error)
+  private def tryParser(string: String, error: DecodingFailure): Xor[DecodingFailure, Instant] = {
+    try { Xor.right(Instant.parse(string)) }
+    catch {
+      case t: IllegalArgumentException =>
+        Xor.left(error)
     }
   }
```

Comment:
Before this PR, acceptable formats for parsing are:
```
yyyy-MM-dd'T'HH:mm:ssZZ
yyyy-MM-dd'T'HH:mm:ss.SSSZZ
```
where ZZ stands for time-zone offset: Z for no offset, +0100, being other examples.


We can choose any of the formatters (and parsers) from:
https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html

Looks like `ISO_OFFSET_DATE_TIME` will match the previous parsing behavior.
- It can parse:
```
2016-06-10T09:47:33.465789+0000
2016-06-10T09:47:33.465789+01:01
```
- But not:
```
2016-06-10T09:47:33.465789+00
2011-12-03T10:15:30+01:00[Europe/Paris]
```